Author: Peter Simon <psimon@gmx.us>, Bjarni Ingi Gislason 
Bugs-Debian: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=745238
Bugs-Debian: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=890044
Description: Fix unreal exit code in manpage.
  Minor other corrections
Last-Updated: 2018-03-20
Forwarded: no

--- a/whiptail.1
+++ b/whiptail.1
@@ -64,11 +64,11 @@
 .B whiptail
 is a program that will let you present a variety of questions or
 display messages using dialog boxes from a shell script. Currently,
-these types of dialog boxes are implemented: 
+these types of dialog boxes are implemented:
 .LP
 .BR yes/no " box," " menu" " box," " input" " box,"
 .BR message " box," " text" " box," " info" " box,"
-.BR checklist " box," " radiolist" " box" " gauge" " box, and"
+.BR checklist " box," " radiolist" " box" ", gauge" " box, and"
 .BR password " box."
 .SH OPTIONS
 .TP
@@ -80,7 +80,7 @@
 an alternate screen.
 .TP
 .B \-\-defaultno
-The dialog box will open with the cursor over the 
+The dialog box will open with the cursor over the
 .BR No " button."
 .TP
 .BI \-\-default\-item " string"
@@ -88,12 +88,12 @@
 Normally the first item in the box is the default.
 .TP
 \fB\-\-fb\fR, \fB\-\-fullbuttons\fR
-Use full buttons. (By default, 
+Use full buttons.  (By default,
 .B whiptail
 uses compact buttons). 
 .TP
 .B \-\-nocancel
-The dialog box won't have a 
+The dialog box won't have a
 .BR Cancel " button".
 .TP
 .BI \-\-yes\-button " text"
@@ -120,12 +120,12 @@
 .B \-\-notags
 Don't display tags in the menu, checklist and radiolist widgets.
 .TP
-.BI \-\-separate\-output
+.B \-\-separate\-output
 For checklist widgets, output result one line at a time, with no
 quoting.  This facilitates parsing by another program.
 .TP
-.BI \-\-output\-fd  " fd" 
-Direct output to the given file descriptor.  Most 
+.BI \-\-output\-fd  " fd"
+Direct output to the given file descriptor.  Most
 .B whiptail
 scripts
 write to standard error, but  error  messages  may  also  be
@@ -141,10 +141,10 @@
 .I backtitle
 string to be displayed on the backdrop, at the top of the screen.
 .TP
-.BI \-\-scrolltext
+.B \-\-scrolltext
 Force the display of a vertical scrollbar.
 .TP
-.BI \-\-topleft
+.B \-\-topleft
 Put window in top-left corner.
 .TP
 \fB\-h\fR, \fB\-\-help\fR
@@ -263,7 +263,7 @@
 .IP "\fB\-\-checklist \fItext height width list-height \fR[ \fItag item status \fR] \fI..."
 .RB "A " checklist " box is similar to a " menu " box in that there are"
 multiple entries presented in the form of a menu.
-You can select and deselect items using the SPACE key.  
+You can select and deselect items using the SPACE key.
 The initial on/off state of each entry is specified by
 .IR status "."
 On exit, a list of the
@@ -286,7 +286,7 @@
 
 .SH NOTES
 whiptail interprets arguments starting with a dash "\-" as being arguments.
-To avoid this, and start some text in, for example, a menubox item, with a 
+To avoid this, and start some text in, for example, a menubox item, with a
 dash, whiptail honours the getopt convention of accepting the special
 argument "\-\-" which means that all following arguments with dashes are to
 be treated verbatim and not parsed as options.
@@ -306,7 +306,7 @@
 .B whiptail
 is exited by pressing the
 .I ESC
-key, the exit status is -1.
+key, the exit status is 255.
 .SH AUTHOR
 Based on the man page for dialog(1) by:
 .LP
